### PR TITLE
Fix example generator

### PR DESCRIPTION
**Pull request by hailin0**
Changes proposed in this pull request:
  - Remove db discovery
  - Fix example output path
---

Before committing this PR, I'm sure that I have checked the following options:
- [x] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [x] I have self-reviewed the commit code.
- [ ] I have (or in comment I request) added corresponding labels for the pull request.
- [ ] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding unit tests for my changes.

### CODE REVIEW

**Comment by totalo** [comment]
hi, @hailin0 Thanks for your attention, here is a small question.

File: examples/shardingsphere-example-generator/src/main/resources/config.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@
 #
 
 output:
-  - ./examples/shardingsphere-example-generator/target/generated-sources/
+  - ./shardingsphere-example-generator/target/generated-sources/
```
Comment (totalo) [inline examples/shardingsphere-example-generator/src/main/resources/config.yaml:19]: This example generation is part of the examples section, so it makes sense to generate examples in the Examples directory.

**Review by totalo — CHANGES_REQUESTED** [review]

**Comment by boyjoy1127** [comment]
Will this issue be completed before June 15 which is the final date of version 5.4.0?

**Comment by boyjoy1127** [comment]
Will this issue be completed before June 15 which is the final date of version 5.4.0?

**Comment by taojintianxia** [comment]
this PR will migrate to milestone 5.4.2

**Comment by strongduanmu** [comment]
Since this pr has long time no response, I will close it.

**Pull request closed without merging**